node/pkg/processor/processor.go

```diff
@@ -216,3 +224,21 @@ func (p *Processor) Run(ctx context.Context) error {
 		}
 	}
 }
+
+func (p *Processor) storePythNetVAA(v *vaa.VAA) {
+	key := fmt.Sprintf("%v/%v", v.EmitterAddress, v.Sequence)
+	p.pythnetVaas[key] = PythNetVaaEntry{v: v, updateTime: time.Now()}
+}
+
+func (p *Processor) getPythNetVAA(v *vaa.VAA) (*vaa.VAA, bool) {
+	key := fmt.Sprintf("%v/%v", v.EmitterAddress, v.Sequence)
+	ret, exists := p.pythnetVaas[key]
+	return ret.v, exists
+}
+
+func (p *Processor) deletePythNetVAA(v *vaa.VAA) bool {
+	key := fmt.Sprintf("%v/%v", v.EmitterAddress, v.Sequence)
+	_, exists := p.pythnetVaas[key]
+	delete(p.pythnetVaas, key)
+	return exists
```

Comment:
nit: we can skip the delete if the entry doesn't exist in the map:

```suggestion
	if _, exists := p.pythnetVaas[key]; exists {
		delete(p.pythnetVaas, key)
		return true
	}
	
	return false
```